packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -164,14 +155,8 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
 
   private _getContentStyles(isSticky: boolean): React.CSSProperties {
     return {
-      backgroundColor: this.props.stickyBackgroundColor || this._getBackground()
-    };
-  }
-
-  private _getStickyContainerStyle(): React.CSSProperties {
-    const { isScrollSynced } = this.props;
-    return {
-      overflow: isScrollSynced ? 'hidden' : 'auto'
+      backgroundColor: this.props.stickyBackgroundColor || this._getBackground(),
```

Comment:
is background color a finite list? Is it going to change frequently? if not, this should just be moved into styling calcs.